ReleaseChecklist.md

```diff
@@ -3,24 +3,27 @@
 ### Requirements
  - [ ] Lauchpad (Ubuntu One) account
  - [ ] gnupg key (has to be version 1, gpg2 won't work) for `your-name@ethereum.org` created and uploaded
- - [ ] Readthedocs account, access to the Solidity project
  - [ ] Write access to https://github.com/ethereum/homebrew-ethereum
 
+### Blog Post
+ - [ ] Create a post on [solidity-blog](https://github.com/ethereum/solidity-blog) and explain some of the new features or concepts.
+ - [ ] Create a post on [solidity-blog](https://github.com/ethereum/solidity-blog) for each important bug.
+
 ### Documentation check
  - [ ] Run `make linkcheck` from within `docs/` and fix any broken links it finds. Ignore false positives caused by `href` anchors and dummy links not meant to work.
 
-### Blog Post
- - [ ] Create a post on https://github.com/ethereum/solidity-blog and explain some of the new features or concepts.
-
 ### Changelog
- - [ ] Sort the changelog entries alphabetically and correct any errors you notice.
- - [ ] Create a commit on a new branch that updates the ``Changelog`` to include a release date.
- - [ ] Run ``./scripts/tests.sh`` to update the bug list.
- - [ ] Create a pull request and wait for the tests, merge it.
+ - [ ] Sort the changelog entries alphabetically and correct any errors you notice. Commit it.
+ - [ ] Copy the changelog into the blog post.
+ - [ ] Update the changelog to include a release date.
+ - [ ] Run ``scripts/update_bugs_by_version.py`` to regenerate ``bugs_by_version.json`` from the changelog and ``bugs.json``.
+ - [ ] Commit changes, create a pull request and wait for the tests. Then merge it.
```

Comment:
This part may need more clarification. I.e. currently we do a separate PR for version and a separate one for changelog. The old text said to `commit on a new branch` and I guess that means two PRs?

I think we would be fine with a single PR and that's what my new version says. Is that ok or are there any good reasons for two PRs?

Reply:
The new sequence is not yet ideal in any case, since it tells you to copy the unreviewed sorted and corrected changelog into the blog post :-).
That'd be *a* reason for doing it in two steps, especially if any change would entail running ``./scripts/test.sh`` locally :-).

But since I don't see much reason for running that locally compared to trusting in CI on the ultimate PR... I guess there's not too much reason to do this in two separate steps. I still for some reason have the feeling that doing things in two steps is cleaner, but I can't come up with an objective reason, so I'm fine with merging it into one step :-).

Reply:
> it tells you to copy the unreviewed sorted and corrected changelog into the blog post :-).

Good point. I moved that to the last point in the section.

> That'd be a reason for doing it in two steps

Right, you have to remember to rerun `scripts/update_bugs_by_version.py` if the changelog is so broken that the script did not even detect the new version. Still, I think that's not a huge risk. I added a warning to make sure the new version is there.

> I still for some reason have the feeling that doing things in two steps is cleaner, but I can't come up with an objective reason, so I'm fine with merging it into one step :-).

For me actually it seems right to have these changes grouped in a single PR. They're all minor, mechanical tweaks that happen because of the new release.

And when doing them or reviewing someone's PR I always check the PR from the previous release to make sure nothing was forgotten. Having just one PR makes this easier.

Reply:
Yeah, as I said, I can't come up with an objective reason not to, so I'm fine with making it one PR :-).